e2e_test/source/README.md

```diff
@@ -0,0 +1,7 @@
+Test in this directory needs some prior setup.
+
+See also `ci/scripts/e2e-source-test.sh`, and `scripts/source`
+
```

Comment:
I think we can provide the steps to run e2e source test here.

Reply:
I haven't yet tried to find an elegant way to run the tests locally. I'm not sure the script can be used locally in current shape.